cfme/services/catalogs/service_catalogs.py

```diff
@@ -9,7 +9,8 @@
 
 order_button = {
     version.LATEST: "//button[@title='Order this Service']",
-    '5.3': "//img[@title='Order this Service']"
+    '5.3': "//img[@title='Order this Service']",
+    '5.4': "//button[@title='Order this Service']"
```

Comment:
We version pick the highest version that's less than or equal to the current appliance version, so this should probably be...
```python
order_button = {
    version.LOWEST: "//img[@title='Order this Service']",
    '5.4': "//button[@title='Order this Service']"
}
```
This will use the img locator for all versions before 5.4, and the button locator for all versions 5.4 and later, including upstream.